velox/docs/functions/spark/datetime.rst

```diff
@@ -25,9 +25,10 @@ These functions support TIMESTAMP and DATE input types.
 
 .. spark:function:: date_add(start_date, num_days) -> date
 
-    Returns the date that is num_days after start_date.
-    If num_days is a negative value then these amount of days will be
+    Returns the date that is ``num_days`` after ``start_date``.
```

Comment:
doc for date_sub says "...the returned date will wrap around..." 

Does this apply to date_add as well? If so, let's include this. Let's make sure that docs for date_add/sub are consistent.